### ISSUE TITLE
Timezone guidance for SIG meetings

### ISSUE THREAD

**Issue by svrnm** [open-telemetry/community#2237]
Since this is a repeated question, I would like to have a dedicated discussion for that and solve it for all SIGs once: with SIGs establishing multiple meeting hours to serve more contributors, we need to consider that scheduling those meetings during Pacific Time has a side-effect around day light saving times. For people outside that timezone, but inside a timezone without day light saving times, the meeting time will adjust twice per year, and for people outside that timezone but inside a timezone with a different day light saving times schedule, the meeting time will change 4 times per. 

As there is no perfect solution for this problem, and going for UTC will just make it worse for everyone, here is my proposal, and we can put this down in a document somewhere:

* All meeting hours between 2pm and 1am UTC hours are express in "PT" (with DST, see image below, the cyan area). 
* All meeting hours between 2am and 1pm UTC  in "UTC +8" (no DST)

---

Image below shows the 2am to 1pm UTC window (https://www.worldtimebuddy.com/?qm=1&lid=100,5391959,1796236&h=100&date=2024-7-25&sln=2-14&hf=1)

![Screenshot 2024-07-25 at 11 05 30](https://github.com/user-attachments/assets/b92a65c5-6c0d-4fd1-9200-300ca222feee)

Image below shows different DST schedules across the world

![Screenshot 2024-07-25 at 11 06 51](https://github.com/user-attachments/assets/8d4e3532-8ad7-4792-8ceb-e1696090fc01)

**Comment by tsloughter** [comment]
Can you explain why UTC+8?

**Comment by svrnm** [comment]
> Can you explain why UTC+8?

There are some reasons for that:
* PT is UTC-8/UTC-7, so there is a 12/11 hours difference
* We already have scheduled some meetings for that timezone
* We already have contributors from there and expect more to come (all of China, Singapore, parts of Australia, Philippines)
* It's one of many potential timezones without a DST, which is the experience of most people across Asia and Australia (see the image above with the gray area)

I used "UTC+8" instead of China Standard Time, Singapore Standard Time, etc because they are equivalent (as of today), since there is no DST.

I used "PT" specifically instead of UTC-8, because of the DST common in the US, where most of the contributors come from. For folks from the EU this is still "good", because while we have a different DST, and there are a few weeks per year where we are off, in 45+ weeks of the year it's just the same time.
